src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,101 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.UserRegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl REGISTRATION_SERVICE;
+    private static User VALID_USER;
+
+    @BeforeAll
+    static void beforeAll() {
+        REGISTRATION_SERVICE = new RegistrationServiceImpl();
+        VALID_USER = User.of("valid_login", "valid_password", 18);
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void validUserCase_Ok() {
```

Comment:
don`t use ok and notOk (https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea)